src/tcache.c

```diff
@@ -788,31 +796,33 @@ tcache_ncached_max_compute(szind_t szind, unsigned current_nhbins) {
 
 static void
 tcache_bin_info_compute(cache_bin_info_t *tcache_bin_info,
-    unsigned tcache_nhbins) {
-	for (szind_t i = 0; i < tcache_nhbins; i++) {
+    unsigned tcache_nbins) {
+	/*
+	 * Compute the values for each bin, but for bins with indices larger
+	 * than tcache_nbins, no items will be cached.
+	 */
+	for (szind_t i = 0; i < TCACHE_NBINS_MAX; i++) {
```

Comment:
FYI only -- keep in mind that later we will also add a way to specify the default per bin ncache_max through malloc_conf.